Metabase Functions.R

```diff
@@ -142,7 +142,7 @@ multiple_summary_PT <- function(data = plhdata_org_clean, by = Org, summary_var,
   attach(data, warn.conflicts = FALSE)
   var_by_Org <- NULL
 table_output <- NULL
-  for (i in levels(Org)){
+  for (i in levels(by)){
```

Comment:
Changed `Org` to `by`:

In the function, we set the parameter names on line 140. Here we set a parameter name as `by` with a parameter value of `Org`.
On line 145 here, I accidentally "hard coded" in `Org` instead of `by`.
This means that it would always read in `Org` rather than whatever we set `by` to equal.

Since we use `by = Org` at the moment this did not bug for us.
But if we wanted to set `by = gender`, it would still run `Org` because we had `Org` here, not `by`.
This fixes that problem. 
Does this make sense?